src/dom/rect.js

```diff
@@ -234,21 +247,63 @@ export default class Rect {
 	}
 
 	/**
+	 * Checks if all properties ({@link #top}, {@link #left}, {@link #right},
+	 * {@link #bottom}, {@link #width} and {@link #height}) are the same as in the other `Rect`.
+	 *
+	 * @param {module:utils/dom/rect~Rect} rect A `Rect` instance to compare with.
+	 * @returns {Boolean} `true` when Rects are equal. `false` otherwise.
+	 */
+	isEqual( anotherRect ) {
+		for ( const prop of rectProperties ) {
+			if ( this[ prop ] !== anotherRect[ prop ] ) {
+				return false;
+			}
+		}
+
+		return true;
+	}
+
+	/**
+	 * Checks whether a `Rect` fully contains another `Rect` instance.
+	 *
+	 * @param {module:utils/dom/rect~Rect} anotherRect
+	 * @returns {Boolean} `true` if contains, `false` otherwise.
+	 */
+	contains( anotherRect ) {
+		const intersectRect = this.getIntersection( anotherRect );
+
+		if ( !intersectRect || !intersectRect.isEqual( anotherRect ) ) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
 	 * Returns a rect of the web browser viewport.
 	 *
 	 * @returns {module:utils/dom/rect~Rect} A viewport rect.
+	 * @param {Boolean} [options.excludeScrollbars] When set `true` the `Rect` will not include
+	 * the scrollbars of the viewport.
 	 */
-	static getViewportRect() {
+	static getViewportRect( options = {} ) {
 		const { innerWidth, innerHeight } = global.window;
-
-		return new Rect( {
+		const rect = new Rect( {
 			top: 0,
 			right: innerWidth,
 			bottom: innerHeight,
 			left: 0,
 			width: innerWidth,
 			height: innerHeight
 		} );
+
+		rect._source = global.window;
+
+		if ( options.excludeScrollbars ) {
+			rect._excludeScrollbarsAndBorders();
```

Comment:
Couldn't this be a public method? It'd make more sense to call it manually, rather than add the `options` param everywhere.